code42.json

```diff
@@ -111,31 +111,14 @@
                     "data_type": "numeric"
                 },
                 {
-                    "column_name": "UserId",
+                    "column_name": "User ID ",
                     "data_path": "action_result.data.*.userId",
                     "data_type": "string",
                     "example_values": [
                         "1"
                     ],
                     "column_order": 1
                 },
-                {
-                    "column_name": "Username",
```

Comment:
Username is already includes via `action_result.parameter` and there is no username on its response anyway.